### ISSUE TITLE
GradCAM varying on multiple calls

### ISSUE THREAD

**Issue by palatos** [sicara/tf-explain#153]
I'm running the GradCAM function on a pretrained model for a given image, and every time I generate the heatmap I get a different one. This happens whether or not I use the guided gradients option. The heatmaps are very different, too.

Is this expected?

**Comment by RaphaelMeudec** [comment]
Let's make sure in v1.0.0 that the parameter `training=True` is passed to the model.predict in tf-explain methods
